prometheus/SOURCES/prometheus.service

```diff
@@ -0,0 +1,19 @@
+# -*- mode: conf -*-
+
+[Unit]
+Description=The Prometheus monitoring system and time series database.
+Documentation=https://prometheus.io
+After=network.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/prometheus
+User=prometheus
+ExecStart=/usr/bin/prometheus \
+          --web.console.libraries=/usr/share/prometheus/console_libraries \
+          --web.console.templates=/usr/share/prometheus/consoles \
+          $PROMETHEUS_OPTS
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=always
+
```

Comment:
Add `StandardOutput` and `StandardError` for [output rediction](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Logging%20and%20Standard%20Input/Output).

Reply:
Oh, thank you. I missed it. Fixed (d2cecc47e49270226b0e0c01753ff49b825fbd98).